pkg/scheduler/apis/config/v1/default_plugins.go

```diff
@@ -60,9 +61,6 @@ func getDefaultPlugins() *v1.Plugins {
 }
 
 func applyFeatureGates(config *v1.Plugins) {
-	if utilfeature.DefaultFeatureGate.Enabled(features.PodSchedulingReadiness) {
-		config.MultiPoint.Enabled = append(config.MultiPoint.Enabled, v1.Plugin{Name: names.SchedulingGates})
```

Comment:
does moving this from the end of the list to the beginning of the list modify scheduler behavior?

Reply:
No, it doesn't. The plugins' order defined in MultiPoint only matter for those ones that belong to the same extension point. `SchedulingGates` is one and the only one plugin that implements the PreEnqueue extension point, so it doesn't intervene with other plugins, and hence doesn't change scheduler's behavior.

Moreover, PreEnqueue is the first step of the whole scheduling workflow, so placing its plugins in the head of Multipoint makes the code more readable, although it doesn't need to.